lage.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     bundle: ['build-tools'],
     clean: [],
     depcheck: ['build-tools'],
+    just: ['build-tools'],
```

Comment:
Given that you are sending calls to execute the prettier and prettier-fix commands this should probably have an entry for prettier and prettier-fix instead of just